c/tskit/tables.c

```diff
@@ -5493,6 +5981,13 @@ tsk_table_collection_has_index(tsk_table_collection_t *self, tsk_flags_t TSK_UNU
         && self->indexes.edge_removal_order != NULL;
 }
 
+// bool
```

Comment:
Why is this commented out?

Reply:
I think this was always commented out? When I uncommented and tried to compile I got this error

```
_tables.c.o' -c ../tskit/tables.c
../tskit/tables.c:5985:1: error: no previous prototype for function 'tsk_edge_table_has_index' [-Werror,-Wmissing-prototypes]
tsk_edge_table_has_index(tsk_table_collection_t *self, tsk_flags_t TSK_UNUSED(options))
^
1 err
```

Reply:
Hmm, it doesn't actually make sense anyway. Must have snuck through at some point. Can you delete it please?